README.md

```diff
@@ -94,16 +94,16 @@ Environment variables:
 
 Example:
 
-  resin-preload --app 123456 --api-token "xxxx..." --img /path/to/resin-os.img
+  balena-preload --app 123456 --api-token "xxxx..." --img /path/to/balena-os.img
 
 ```
 
 If using environment variables, this program requires the following options to be set:
 
   * `IMAGE`: The path to the OS image you downloaded for the `APP_ID` you want to target.
   * `APP_ID`: ID of the App that will be assigned to the device. It can be extracted from the URL in the Resin dashboard,
-     for instance `https://dashboard.resin.io/apps/2167` means the `APP_ID` is `2167`.
-  * `API_TOKEN`: Authentication token for Resin, taken from the [preferences page](https://dashboard.resin.io/preferences/details). 
+     for instance `https://dashboard.balena.io/apps/2167` means the `APP_ID` is `2167`.
+  * `API_TOKEN`: Authentication token for Resin, taken from the [preferences page](https://dashboard.balena.io/preferences/details).
```

Comment:
@zvin I believe it will be dashboard.balena-cloud.com

Reply:
updated

Reply:
@shaunmulligan will it be api.balena-cloud.io and forums.balena-cloud.io too ?

Reply:
@zvin the api will be `api.balena-cloud.com` not the `.io` the forums i'm not sure yet, i think it will be forums.balena.io